content/en/docs/hertz/getting-started/_index.md

```diff
@@ -1,49 +1,60 @@
 ---
-title: "Getting Started"
-linkTitle: "Getting Started"
+title: 'Getting Started'
+linkTitle: 'Getting Started'
 weight: 2
 description: >
 ---
 
 ## Set Up Golang Development Environment
+
 1. If you haven't set up your Golang Environment, you can refer to [Golang Install](https://golang.org/doc/install).
 2. We recommend you to use the Golang latest version, or make sure it's >= v1.15. You can choose to use the earlier versions, but the compatibility and stability are not assured.
 3. Make sure the go mod support is on (for Golang versions >= v1.15, it is on by default).
 
->Currently, Hertz supports Linux, macOS, and Windows systems.
+> Currently, Hertz supports Linux, macOS, and Windows systems.
 
 ## Quick Start
+
 After you have prepared the Golang environment, the chapter will help you to quickly get familiar with Hertz.
 
 ### Install the commend tool of hz
+
 First, you need to install the commend tool hz which is used in this chapter
+
 1. Confirm the `GOPATH` environment has been defined correctly (For example `export GOPATH=~/go`)
-and the `$GOPATH/bin` has been added to `PATH` environment (For example `export PATH=$GOPATH/bin:$PATH`);
-Attention, do not set a directory in which you have no read and write permission as `GOPATH`.
+   and the `$GOPATH/bin` has been added to `PATH` environment (For example `export PATH=$GOPATH/bin:$PATH`);
+   Attention, do not set `GOPATH` to a directory that the current user does not have read/write access to.
```

Comment:
no need to add space before each line

Reply:
prettier... I use prettier format my code.

Reply:
> no need to add space before each line

which formatter do you use? we need to unify...